frontend/src/constants/DashboardConstants.ts

```diff
@@ -1,2 +0,0 @@
-const UPCOMING_WEEK_LIMIT = "2";
```

Comment:
why didnt you just make this 0 rather than all the deletions? What if we need weekLimit in the future even though its not planned?